src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -3577,10 +3577,10 @@ internal static bool IsTypeParameterizedProperty(Type t)
             {
                 case null: return null;
                 case PropertyCacheEntry cacheEntry when lookingForProperties:
-                {
-                    var isHidden = cacheEntry.member.GetCustomAttributes(typeof(HiddenAttribute), false).Any();
-                    return new PSProperty(cacheEntry.member.Name, this, obj, cacheEntry) { IsHidden = isHidden } as T;
-                }
+                    {
```

Comment:
Seems odd to have the brace indented particularly since the brace isn't even needed

Reply:
Fixed.